frontend/src/components/UserStoryDescriptions.vue

```diff
@@ -95,43 +99,50 @@ export default Vue.extend({
   watch: {
     initialStories() {
       this.userStories = this.initialStories as Array<{
-        jiraId: string | null;
+        id: string | null;
         title: string;
         description: string;
         estimation: string | null;
         isActive: boolean;
+        position: number;
       }>;
     },
   },
   created() {
     this.userStories = this.initialStories as Array<{
-      jiraId: string | null;
+      id: string | null;
       title: string;
       description: string;
       estimation: string | null;
       isActive: boolean;
+      position: number;
     }>;
   },
   methods: {
+    valueChanged(idx, {markdown}) {
+      this.userStories[idx].description = markdown;
+    },
```

Comment:
Perhaps you can already call `this.publishChanges(index);` here, so there is no need to press the button to update the user story for all members.